.github/CODE_OF_CONDUCT.md

```diff
@@ -56,3 +56,39 @@ Original text courtesy of the Speak Up! project and Django Project.
 ## QUESTIONS?
 
 If you have questions, please see the FAQ. If that doesn’t answer your questions, feel free to email conduct@js.foundation.
+
```

Comment:
`conduct@js.foundation` probably is outdated, should be something with openjs.

Reply:
I'll update to `report@lists.openjsf.org`